app/views/resources/_collaborators.html.erb

```diff
@@ -13,4 +13,14 @@
       <% end %>
     </div>
   <% end %>
+
+  <% if resource.group_resources.present?  && ROLLOUT.active?(:collaborator_groups) %>
```

Comment:
Maybe flip the order of these for a fast fail when the feature is not enabled. No trip to the DB to look up non-existent `group_resources` when the feature is turned off.